src/applications/pact-act/containers/questions/agent-orange/AgentOrange-2-2-1-A.jsx

```diff
@@ -78,11 +78,11 @@ const mapDispatchToProps = {
 
 Orange221A.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setOrange221A: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.